examples/typescript-plugin/src/services/index.ts

```diff
@@ -0,0 +1,14 @@
+import { StrapiInterface } from '@strapi/strapi';
+
+export type ServiceMap = { [key: string]: (options: { strapi: StrapiInterface }) => object };
+
+const services: ServiceMap = {
+  foo: ({ strapi }) => ({
```

Comment:
here it's actually ({ strapi }) => ({ foo() {} }) . Strapi isn't passed in each method

Reply:
Foo isn't the method though, it's the service.

We're looking for smth like this, right?

```js
module.exports = () => ({
  services: {
    serviceA: ({ strapi }) => ({
      method1: () => 'foo'
    })
  }
})
```

Reply:
Indeed, I've extracted the foo service into its own file.